docs/openj9_support.md

```diff
@@ -132,9 +132,9 @@ OpenJDK 11 binaries are expected to function on the minimum operating system lev
 | Windows Server 2016                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
 | Windows Server 2019                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
 
-| macOS                                     | x64                                                                                  |
-|-------------------------------------------|--------------------------------------------------------------------------------------|
-| OS X 10.11+                               | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| macOS         | x64                                                                                  | AArch64                                                                                                |
+|---------------|--------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| OS X 10.11+   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
```

Comment:
The AArch64 build does not support OS X 10.15.7 and older.

Reply:
Please remove the row for macOS 10.15.7+, and add rows for macOS 11 and 12 instead.
For Java 11 and AArch64, 
- OS X10.11+ has to be no.
- macOS 11 has to be yes.
- macOS 12 has to be yes.

Reply:
Done.

Reply:
LGTM.